packages/frontend-shared/src/gql-components/error/BaseError.vue

```diff
@@ -23,8 +23,9 @@
             variant="outline"
             data-testid="error-retry-button"
             :prefix-icon="RestartIcon"
-            prefix-icon-class="icon-dark-indigo-500"
-            @click="retry?.(baseError.id)"
+            :prefix-icon-class="`${retrying && 'animate-spin'} icon-dark-indigo-500`"
```

Comment:
I think it's more common to use the syntax made popular by the `classes` library, often seen in React projects, using the object syntax. If the value is `true`, the class is rendered. Docs: https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes

```js
:prefix-icon-class="{ 'animate-spin': retrying, icon-dark-indigo-500` }"
```
It looks like `BaseError` would need to be updated:

```js
const props = defineProps<{
  prefixIconClass?: string | Record<string, string | boolean>
}>()
```

Seems like a quick win.